css/main.css

```diff
@@ -1,13 +1,29 @@
 @import url("https://fonts.googleapis.com/css?family=Montserrat&display=swap");
 @import url("https://fonts.googleapis.com/css?family=Mansalva|Pacifico&display=swap");
 @import url("https://fonts.googleapis.com/css?family=Open+Sans&display=swap");
+@import url('https://fonts.googleapis.com/css?family=Long+Cang|Roboto&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Be+Vietnam&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Shadows+Into+Light&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Amatic+SC&display=swap');
```

Comment:
We are loading too many fonts here. We should stick to not more than 3 otherwise the website will become slow, and from what I can see we aren't using these fonts either. Remove the ones you aren't using please.

Reply:
I have removed it. Please merge.